{{cookiecutter.repo_name}}/README.rst

```diff
@@ -12,12 +12,118 @@ The XBlock SDK Workbench, including this XBlock, will be available on the list o
 Translating
 -----------
 
+Internationalization (i18n) is when a program is made aware of multiple languages.
+Localization (l10n) is adapting a program to local language and cultural habits.
+
 Use the locale directory to provide internationalized strings for your XBlock project.
+For more information on how to enable translations, visit the
+[Open edX XBlock tutorial on Internationalization](http://edx.readthedocs.org/projects/xblock-tutorial/en/latest/edx_platform/edx_lms.html).
+
+This cookiecutter template uses [django-statici18n](https://django-statici18n.readthedocs.io/en/latest/)
+to provide translations to static javascript using `gettext`.
+
+The included Makefile contains targets for extracting, compiling and validating translatable strings.
+The general steps to provide multilingual messages for a Python program (or an XBlock) are:
+
+1. Mark translatable strings.
+2. Run i18n tools to create raw message catalogs.
+3. Create language specific translations for each message in the catalogs.
+4. Use `gettext` to translate strings.
```

Comment:
Would it make sense to merge this with `1` since you perform this while marking the strings translatable?

Reply:
The steps were based on [python's i18n docs.](https://docs.python.org/3/library/gettext.html#internationalizing-your-programs-and-modules)

`1` and `4` are slightly different. In `1` we use `gettext` to mark the strings and, although `gettext` is also used to translate the strings, in `4` `gettext` is used to lookup the translation.
In 1 we do `gettext("string to translate.")` while in 4 we are referring to what Django actually does when translating messages from the compiled files.

Maybe it would be better to remove `4` here since it's something the user doesn't necessarily has to worry about.